src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -2815,15 +2815,12 @@ public static Town newTown(TownyWorld world, String name, Resident resident, Coo
 				// This plot is in the regeneration queue.
 				TownyRegenAPI.removeFromActiveRegeneration(plotChunk); // just claimed so stop regeneration.
 				TownyRegenAPI.removeFromRegenQueueList(townBlock.getWorldCoord()); // Remove the WorldCoord from the regenqueue.
+				TownyRegenAPI.addPlotChunkSnapshot(plotChunk); // Save a snapshot.

```

Comment:
This isn't done here because we would be overwriting another snapshot that was made earlier, replacing it with not-original wilderness.